README.md

```diff
@@ -6,3 +6,39 @@ Notebook samples are grouped per kernel (sentinelhub, openeo and geo).
 ## Automated testing
 If you want your sample to be automatically tested, add it to the `.tests` file. A GitHub action will pull the
 Jupyterlab container and automatically test the listed notebook samples.
+
+## Setting Up Pre-commit Hooks
```

Comment:
Setting up pre-commit hooks is pretty specific for contributors, I would not put this so prominently  at the top of the README.
I think it should be just a small not under a "For contributors" section